# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lcap/install.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	Sun Mar 24 16:32:52 2002
+++ ./Makefile	Sun Mar 24 16:33:55 2002
@@ -29,14 +29,16 @@
 LDFLAGS = -s
 
 FILES = COPYING INSTALL Makefile README lcap.c lcap.spec lcap.8
+
+prefix=/usr
   
 all:		lcap
 
 lcap:		lcap.o
 
 install:	lcap
-		install -m700 -D lcap $(prefix)/sbin/lcap
-		install -m444 -D lcap.8 $(prefix)/usr/man/man8/lcap.8
+		install -m700 -D lcap /sbin/lcap
+		install -m444 -D lcap.8 $(prefix)/man/man8/lcap.8
 
 rpms:
 		$(MAKE) dist
